tests/integration/test_manifest_validation.py

```diff
@@ -0,0 +1,486 @@
+"""
+This module is responsible for running through the "Manifest Validation" portion of
+the schematic API test plan found here: <https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/3055779846/Schematic+API+test+plan>.
+"""
+
+import os
+import json
+import pytest
+import requests
+from tests.conftest import Helpers, testing_config, ConfigurationForTesting
+
+
+SYNAPSE_PAT = os.getenv("SYNAPSE_PAT")
+if not SYNAPSE_PAT:
+    raise EnvironmentError("SYNAPSE_PAT environment variable is not set.")
+
+HEADERS = {"Authorization": f"Bearer {SYNAPSE_PAT}"}
+EXAMPLE_SCHEMA_URL = "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld"
+
+@pytest.fixture(scope="module")
+def setup_api(testing_config: ConfigurationForTesting) -> str:
+    url = testing_config.schematic_api_server_url
+
+    # Make a request to the API and make sure it is responsive
+    response = requests.get(url)
+
+    assert response.status_code == 200, f"Failed to connect to API: {response.text}"
+    return url
```

Comment:
Some complexity here is shown in this commit that I made to other parts of the code:
https://github.com/Sage-Bionetworks/schematic/pull/1512/commits/21d95f5bb60d8ca94e52c0332a36b20679d5c405

Basically in 1 case: When the API server is running on localhost separately from the integration test run, OR, deployed to DEV we can use the requests library.

In the second case: We need to continue using the `flask_client: FlaskClient` pytest fixture. This is for if the integration test is running in the GH pipeline, or we directly run the tests with pytest without starting a local schematic API server.

Unfortunately, the way to invoke `.get()` and retrieves the response differs between the 2 libraries, however, the code I linked shows show I am doing this.


To test out one or the other will depend on what you've set in your `.env` file. The levers that can be flipped are detailed here: https://github.com/Sage-Bionetworks/schematic/blob/21d95f5bb60d8ca94e52c0332a36b20679d5c405/env.example#L15-L30